_jobs/positions/__QA-testing-engineer.md

```diff
@@ -0,0 +1,54 @@
+
+This vital position has the potential to create positive change inside our organization and across City of Austin services. The person in this role will be instrumental in shaping the future of digital, accessible, and equitable city services. We're looking for someone passionate, creative, resourceful, civic-minded, and impact-driven. This may involve a wide range of activities, from helping solidify existing QA, testing, and release protocols to helping write or review unit tests and set up cross-browser testing.
+
+
+## Salary
+
+$75,000-$86,000, commensurate with experience
+
+## What you’ll do:
+
+* Take the lead in collaborating to build and maintain effective testing practices when appropriate, including:
+  * Designing and implementing unit tests. 
+  * Evaluating test coverage. 
+  * Setting up automated browser testing. 
+  * Participating in collaborative code reviews with the rest of development staff. 
+* Serve as a core contributor on multi-disciplinary project teams to improve city services. 
+* Contribute to Open Source Software projects using Github for collaboration and code review. For a flavor of our current stack, we maintain projects that use:
+    * React.js (Javascript), Python (Django), Node.js, Jekyll (Ruby), Apollo & GraphQL, Docker, Heroku, and AWS.
+* Verify that new features, stories, and bug fixes are articulated with testing in mind. 
+* Be an advocate for testability early in the process of determining user stories and project requirements. 
+* Assist in scoping and prototyping activities for future design and development projects. 
+* Work closely with your fellow developers as well as content and design teams to understand new features and projects, watching out for ways their development can be facilitated by robust and efficient QA. 
+* Help manage product releases and related regression testing. 
+* Build and share knowledge of web standards and best practices related to testing and QA. 
+* Engage with the wider web development community, identifying best practices we can adopt and sharing what we're building and learning. 
+* Commit to making what we build fully accessible to the people we serve.
+
+
+## What we're looking for
+
+* A passion for public service and a commitment to doing the hard work that’s necessary to improve city services
+* 3+ years of professional experience developing large-scale applications and services, including at least 2 years of experience directly with testing, QA, and automation
+* Knowledge and experience with testing frameworks in Javascript and Python
+* Knowledge and experience with browser testing frameworks such as Selenium or Cypress
+* Knowledge of lean and agile development methodologies, including expertise in teaching when and how to use specific specific approaches and deliverables
```

Comment:
```suggestion
* Knowledge of lean and agile development methodologies, including expertise in teaching when and how to use specific specific approaches and deliverables. 
```

Reply:
I think we can remove the part about teaching when to use them. knowledge of is enough for this role.